aws-codeguruprofiler-profilinggroup/aws-codeguruprofiler-profilinggroup.json

```diff
@@ -45,6 +45,16 @@
         }
       }
     },
+    "ChannelUris": {
+      "description": "The Amazon Resource Name (ARN) of the specified SNS arn",
```

Comment:
I suggest renaming to `The Amazon Resource Name (ARN) of the specified SNS Topic.`